deephub/resources/remote.py

```diff
@@ -124,12 +125,13 @@ def load_resources_from_yaml(filepath=None):
     if filepath is None:
         filepath = Path(__file__).parent / 'remote.yaml'
 
```

Comment:
I think we should add an else here with a raise that the path does not exists.

Reply:
I added this because if you pip install the pck then the wheel contains only .py files. So if you run deep it tries to find remote.yaml which doesn't exists and breaks. So when you pip install deephub it's for sure doesn't include any resources so it doesn't make sense to break or raise an exception.

Reply:
And what will happen if after I have download the resources (products.yaml), rename this file (products.yaml)?

Reply:
I mean what will happen if I give an input that does not exists in this function?

Reply:
write I agree.. I will change this to take this into account..